### DISCUSSION TITLE
posts/2019-02-22/reverse-proxy-and-intranet-through/

### DISCUSSION THREAD

**Discussion by giscus** [tobyqin/tobyqin.github.io#6] [Comments]
# posts/2019-02-22/reverse-proxy-and-intranet-through/

学习一下正向代理和反向代理。

https://tobyqin.cn/posts/2019-02-22/reverse-proxy-and-intranet-through/

**Comment by hsiunien** [comment]
确实，花生壳越来越不好用，如果是正规企业谁还会去用内网穿透呢？自己运维配置一下不就好了。
